cmd/test/test.go

```diff
@@ -0,0 +1,228 @@
+)
+
+const (
+	OutputAppstudio = "appstudio"
+)
+
+const testDesc = `
+The 'ec test' command is a thin wrapper for the 'conftest test' command.
+
+This command tests your configuration files using the Open Policy Agent.
+
+The test command expects one or more input files that will be evaluated
+against Open Policy Agent policies. Directories are also supported as valid
+inputs.
+
+Policies are written in the Rego language. For more
+information on how to write Rego policies, see the documentation:
+https://www.openpolicyagent.org/docs/latest/policy-language/
+
+The policy location defaults to the policy directory in the local folder.
+The location can be overridden with the '--policy' flag, e.g.:
+
+	$ ec test --policy <my-directory> <input-file(s)/input-folder>
+
+Some policies are dependant on external data. This data is loaded in separately
+from policies. The location of any data directory or file can be specified with
+the '--data' flag. If a directory is specified, it will be recursively searched for
+any data files. Right now any '.json' or '.yaml' file will be loaded in
+and made available in the Rego policies. Data will be made available in Rego based on
+the file path where the data was found. For example, if data is stored
+under 'policy/exceptions/my_data.yaml', and we execute the following command:
+
+	$ ec test --data policy <input-file>
+
+The data is available under 'import data.exceptions'.
+
+The test command supports the '--output' flag to specify the type, e.g.:
+
+	$ ec test -o table -p examples/kubernetes/policy examples/kubernetes/deployment.yaml
+
+Which will return the following output:
++---------+----------------------------------+--------------------------------+
+| RESULT  |               FILE               |            MESSAGE             |
++---------+----------------------------------+--------------------------------+
+| success | examples/kubernetes/service.yaml |                                |
+| warning | examples/kubernetes/service.yaml | Found service hello-kubernetes |
+|         |                                  | but services are not allowed   |
++---------+----------------------------------+--------------------------------+
+
+By default, it will use the regular stdout output. For a full list of available output types, see the of the '--output' flag.
+
+The test command supports the '--update' flag to fetch the latest version of the policy at the given url.
+It expects one or more urls to fetch the latest policies from, e.g.:
+
+	$ ec test --update opa.azurecr.io/test
+
+See the pull command for more details on supported protocols for fetching policies.
+
+When debugging policies it can be useful to use a more verbose policy evaluation output. By using the '--trace' flag
+the output will include a detailed trace of how the policy was evaluated, e.g.
+
+	$ ec test --trace <input-file>
+`
+
+// NewTestCommand creates a new test command.
+func NewTestCommand() *cobra.Command {
```

Comment:
This could be `newTestCommand` but I'm leaving it like this so the diff is easier to read...

Reply:
Capitalization invokes exportation, if that's not the intent, this should be left lowercased.

Reply:
Yeah, I did it on purpose since it reduces the diff between the conftest file and our file. I could maybe add a comment to explain.

Reply:
I think it would be better to follow the idioms of Go and not capitalize function names if they're not intended to be exported.

Reply:
Yeah ok.

Reply:
Done in latest revision.